<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLoginTitle</class>
 <widget class="QWidget" name="WidgetLoginTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lbLogo">
     <property name="text">
      <string>Logo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbTitle">
     <property name="font">
      <font>
       <family>BM Jua</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(11, 54, 79);</string>
     </property>
     <property name="text">
      <string>Foxcloud</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="btnSetServer">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnHide">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnClose">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
